### PR TITLE
GitHub message error paths

### PR DESCRIPTION
**Pull request by TerrenceMcGuinness-NOAA**
# Description

Add feature to message paths to error logs of failed experiments to GitHub Messaging the PR from Jenkins.
This will be tested from forked branch in another Repo

### CODE REVIEW

File: ci/Jenkinsfile [hunk 1]
```diff
@@ -155,6 +155,12 @@ pipeline {
                                         sh(script: "${HOMEgfs}/ci/scripts/utils/ci_utils_wrapper.sh cancel_all_batch_jobs ${HOME}/RUNTESTS")
                                         ws(HOME) {
                                             if (fileExists('RUNTESTS/error.logs')) {
+                                                def error_logs = sh(script: "cat RUNTESTS/error.logs", returnStdout: true).trim()
+                                                try {
+                                                    pullRequest.comment("Experment ${Case} failed on ${Machine}\n\nError logs:\n\n${error_logs}")
```
Comment (aerorahul) [inline ci/Jenkinsfile]:
```suggestion
                                                    pullRequest.comment("Experiment ${Case} failed on ${Machine}\n\nError logs:\n\n${error_logs}")
```

**Review by aerorahul — APPROVED** [review]
looks good.

**Review by WalterKolczynski-NOAA — APPROVED** [review]
Looks good, but what we really want to do is upload the artifact to GH.

**Comment by TerrenceMcGuinness-NOAA** [comment]
@WalterKolczynski-NOAA Upload archives to GitHub: Yes, I have the instructions for that, requires a few plugins and specific admin configurations.  Need to code it in and test on the the development controler.

**Pull request merged**